test/TestLoopFuser.cxx

```diff
@@ -143,7 +135,8 @@ GPU_TEST(TestPacker, packFixedRangeMacro) {
 
    care::gpuDeviceSynchronize(__FILE__, __LINE__);
 
-   // pack should  not have happened yet so
+#ifdef CARE_GPUCC
```

Comment:
Maybe add comment by CARE_GPUCC protection is needed here

Reply:
Maybe add comment by CARE_GPUCC protection is needed here and also below in other places where you did this, just for clarity

Reply:
Updated the comment (now it specifically mentions the device, so I think that should be sufficient).